intellij-ide.md

```diff
@@ -185,6 +185,21 @@ your debugging session:
 1. In the Debug panel at the bottom, click the **Full Application Restart button**
 (shown with an icon containing a circular arrow).
 
+## Tips and tricks
+
+Keyboard shortcuts:
+  * JetBrains has a good [Keyboard Shortcuts You Cannot Miss](https://www.jetbrains.com/help/idea/keyboard-shortcuts-you-cannot-miss.html) overview
+  * Customize the shortcuts to your preferences in **Preferences > Keymap**
```

Comment:
we usually recommend to learn the default shortcuts rather than customize them :)